pyseir/cli.py

```diff
@@ -144,14 +165,23 @@ def _build_all_for_states(
         root.info("Only executing for states. returning.")
         return
 
+    # Separating the Infection Rate Calculations
+    infection_rate_metric_df = _generate_infection_rate_metric()
```

Comment:
I think this should take in the input regions - otherwise you won't be able to run on subsets locally

Reply:
done